src/transformers/models/layoutlm/configuration_layoutlm.py

```diff
@@ -125,3 +129,86 @@ def __init__(
             **kwargs,
         )
         self.max_2d_position_embeddings = max_2d_position_embeddings
+
+
+class LayoutLMOnnxConfig(OnnxConfig):
+
+    DEFAULT_MAX_2D_POSITION = 1000
```

Comment:
I'm not very familiar with this model, so asking just in case: 

Would it be interesting to change this default value to a config instance variable? In order for a user to export the model easily with a different version of this parameter?

cc @LysandreJik @NielsRogge

Reply:
@mfuntowicz  You are right. The config already has an instance variable for this. I have updated the class to use that instead of the class variable I had used earlier. Thanks for pointing it out!

Reply:
Hi, @mfuntowicz @LysandreJik @NielsRogge any update about this?

Reply:
Sorry, only seeing this now. I guess you'll need to open a new PR and close this one.

Reply:
@nishprabhu mind opening a new PR?

Reply:
Yes, I'll close this one and open a new one. @NielsRogge @mfuntowicz @LysandreJik